### PR TITLE
Duplication of the same resource

### PR DESCRIPTION
**Pull request by amandeepmittal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1046,8 +1047,6 @@ A collection of awesome things regarding React ecosystem.
 * [Isomorphic Development with JavaScript](https://www.manning.com/books/isomorphic-development-with-javascript)
 * [React in Action](https://www.manning.com/books/react-in-action)
 * [React Quickly](https://www.manning.com/books/react-quickly)
-* [The Road to learn React](https://roadtoreact.com/course-details?courseId=THE_ROAD_TO_LEARN_REACT)
```
Comment (Haroenv) [inline README.md]: where are they duplicated?

**Comment by amandeepmittal** [comment]
Both of the resources lead to the same website, IMO.

File: README.md (hunk 1 above)
Comment (paulmelnikow) [reply README.md]: Besides these two, there is a different link to "The Road to Learn React" under books. That seems sufficient to me.

File: README.md [hunk 2]
```diff
@@ -921,7 +921,8 @@ A collection of awesome things regarding React ecosystem.
 * [express-graphql - Create a GraphQL HTTP server with **Express**](https://github.com/graphql/express-graphql)
 
 #### Database Integration
-* [graphql-bookshelf - Some help defining GraphQL schema around BookshelfJS models](https://github.com/brysgo/graphql-bookshelf)
+* [graphql-bookshelf - Some help defining GraphQL schema around 
+helfJS models](https://github.com/brysgo/graphql-bookshelf)
```
Comment (paulmelnikow) [inline README.md]: Can you clean up this line break?

**Pull request merged**